frontend/packages/integration-tests-cypress/support/index.ts

```diff
@@ -71,3 +71,13 @@ export const editKind = (kind: string, humanizeKind: boolean) =>
   actionOnKind(actions.edit, kind, humanizeKind);
 export const deleteKind = (kind: string, humanizeKind: boolean) =>
   actionOnKind(actions.delete, kind, humanizeKind);
+
+export const create = (obj) => {
+  const filename = [
+    Cypress.config('screenshotsFolder'),
+    `${obj.metadata.name}.${obj.kind.toLowerCase()}.json`,
```

Comment:
Hi @rhamilto, `Cypress.config('screenshotsFolder')` refers to `/frontend/gui_test_screenshots/cypress/screenshots/` the location where the cypress screenshots go.  You probably want to create these in `/frontend/gui_test_screenshots/` where the other result .json files get created.  Probably not a big issue as I see you are deleting these files after the test runs.

Reply:
This seems to work for me:
```
Cypress.config('screenshotsFolder')
    .toString()
    .replace('/cypress/screenshots', '')
```

Reply:
Why not just hard code the path rather than manipulating the config value?  For example, 
```
const filename = [
    '/frontend/gui_test_screenshots',
    `${obj.metadata.name}.${obj.kind.toLowerCase()}.json`,
```

Reply:
Because we `cd` to the OLM package/dir and that will write the file in `packages/operator-lifecycle-manager/integration-tests-cypress/frontend/gui_test_screenshots`!   Each package (devconsole, olm, ceph) has their own `cypress.json` config file with a `Cypress.config('screenshotsFolder')` which points to the console's gui  `gui_test_screenshots/cypress/screenshots` dir.  Ie. OLM it is `"screenshotsFolder": "../../../gui_test_screenshots/cypress/screenshots"`, so trimming off the `/cypress/screenshots` writes files to the correct place.

Or, If you just do `const filename = './${obj.metadata.name}.${obj.kind.toLowerCase()}.json';`   That should write the files to `packages/operator-lifecycle-manager/integration-tests-cypress/` which should be ok as you are cleaning them up after tests.  I guess the only issue would be if tests fails and cleanup doesn't happen, then you'll have these files left in the location you decide.  (console's `gui_test_screenshots/cypress/screenshots` is already in .gitignore)

Reply:
Gotcha.  Thanks for clarifying.  Updated with your initial suggestion.